go/form.templ

```diff
@@ -0,0 +1,30 @@
+package popui
+
+import (
+	"github.com/invopop/popui/go/classes"
+	"github.com/invopop/popui/go/props"
+)
+
+templ Form(p props.Form) {
```

Comment:
I think it'd be great to add comments to these sections so that its clear what the component is meant to be used for.

```go
// Form provides a wrapper around a set of form components and ensures
// that the fields will be laid out correctly.
templ Form(p props.Form) {
```